specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/locations.json

```diff
@@ -322,14 +364,14 @@
             "$ref": "#/definitions/RegionsCapability"
           }
         },
-        "vmSizes": {
+        "vmsizes": {
           "description": "The virtual machine sizes.",
           "type": "object",
           "additionalProperties": {
             "$ref": "#/definitions/VmSizesCapability"
           }
         },
-        "vmSize_filters": {
+        "vmsize_filters": {
```

Comment:
The same with the property "vmsizes", I know this property should be changed to vmSizeFilters, but now in the backend, the response is vmsize_filters. I have created a work item to track this.
Our service will GA a new api version in April, in the new api version, I will fix this issue.
If it is ok, I hope this can check in. @JeffreyRichter You opinion is important for us, I will follow your suggestion. How do you like?

Reply:
New api-versions of a service MUST be backwards compatible which means that this must NOT be fixed in a new api-version. This must remain incorrect forever. The swagger must accurately reflect the service behavior and so the swagger must remain incorrect forever too.